.circleci/config.yml

```diff
@@ -281,49 +228,45 @@ jobs:
             cp _build/test/dialyxir*.plt plts/
             cp ~/.mix/dialyxir*.plt plts/
       - save_cache:
-          key: v1-plt-cache-{{ ".tool-versions" }}-{{ checksum "mix.lock" }}
+          key: v2-plt-cache-{{ ".tool-versions" }}-{{ checksum "mix.lock" }}
           paths:
             - plts
       - save_cache:
-          key: v1-plt-cache-{{ ".tool-versions" }}-{{ checksum "mix.exs" }}
+          key: v2-plt-cache-{{ ".tool-versions" }}-{{ checksum "mix.exs" }}
           paths:
             - plts
       - save_cache:
-          key: v1-plt-cache-{{ ".tool-versions" }}
+          key: v2-plt-cache-{{ ".tool-versions" }}
           paths:
             - plts
 
       - run: mix dialyzer --format short --halt-exit-status
 
-  release:
-    docker:
-      # Ensure .tool-versions matches
-      - image: omisegoimages/elixir-omg-circleci:v1.8-20190129-02
-        environment:
-          MIX_ENV: dev
-    working_directory: ~/repo
+  publish_child_chain:
+    executor: metal_child_chain
+    steps:
+      - checkout
+      - run: make docker-child_chain CHILD_CHAIN_IMAGE_NAME=$CHILD_CHAIN_IMAGE_NAME
+      #- run: sh .circleci/ci_publish.sh
```

Comment:
is there a plan for the commented lines here or just some leftovers to be deleted? If the former, maybe worth to comment

Reply:
yep, that comes in as part 2 when I setup deployment!